### DISCUSSION TITLE
fail2ban-action for MS SQL Server

### DISCUSSION THREAD

**Discussion by sachtouris** [fail2ban/fail2ban#3237] [General]
So i have created a bash script (https://github.com/sachtouris/fail2ban-action-mssql) that gets the banned ips and adds them to your MS SQL Server. My idea is if you could add to your jail.conf the [mssqld] for compatibillity with MS SQL. The port that is using is 1433 usually if its not changed and the logpath=/var/opt/mssql/log/errorlog

**Comment by sebres** [comment]
> My idea is if you could add to your jail.conf the [mssqld] for compatibillity with MS SQL

There is another jail [mssql-auth](https://github.com/fail2ban/fail2ban/blob/f81f85569d88da74d76f286b431dc54d0ae00748/config/jail.conf#L800) and I don't see why we'd need still one jail `mssqld` here.
Also we don't basically add any custom actions to any jail by default (if it is not expected for some reason), if someone'd need that, it is simply possible within `jail.local` for `mssql-auth` or whatever else jail.

**Reply by sachtouris** [reply]
Oh i didnt check on github if you had a mssql-auth i propably have older version of jail2ban|(although i installed it via ubuntus apt) and it isnt contained on jail.conf or the mssql filter doesnt exist.
